clash-prelude/src/Clash/Signal/Delayed/Internal.hs

```diff
@@ -123,7 +123,7 @@ dfromList_lazy = coerce . fromList_lazy
 --     mac' :: 'DSignal' dom 0 Int -> 'DSignal' dom 0 Int -> 'DSignal' dom 0 Int
 --          -> ('DSignal' dom 0 Int, 'DSignal' dom 1 Int)
 --     mac' a b acc = let acc' = a * b + acc
---                    in  (acc, 'delay' clk rst en ('singleton' 0) acc')
+--                    in  (acc, 'Clash.Explicit.Signal.Delayed.delay' 0 en clk rst acc')
```

Comment:
`Clash.Explicit.Signal.Delayed.delay` doesn't exist.
```
    in  (acc, 'Clash.Explicit.Signal.Delayed.delayedI' 0 en clk rst acc')
```

Reply:
Right, and it takes clk/rst/en first too